src/System.Drawing.Design.Primitives/tests/Design/ToolboxItemCollectionTests.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+namespace System.Drawing.Design.Primitives.Tests.System.Drawing.Design
+{
+    public class ToolboxItemCollectionTests
+    {
+        private ToolboxItemCollection CreateToolboxItemCollection()
+        {
+            ToolboxItem[]  tools = { new ToolboxItem(), new ToolboxItem() };
+
+            return new ToolboxItemCollection(tools);
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void Contains_StateUnderTest_ExpectedBehavior()
+        {
+            // Arrange
+            var unitUnderTest = CreateToolboxItemCollection();
+            ToolboxItem value = unitUnderTest[0];
+
+            // Act
+            var result = unitUnderTest.Contains(
+                value);
+
+            // Assert
```

Comment:
Nit: I understand it's a common pattern, but even if we follow Arrange/Act/Assert, we generally don't include the comments stating that in our corefx unit tests.